storage/innobase/dict/dict0dd.cc

```diff
@@ -1642,7 +1642,6 @@ void dd_table_close(dict_table_t *table, THD *thd, MDL_ticket **mdl,
 
 #ifndef UNIV_HOTBACKUP
   if (mdl != nullptr && *mdl != nullptr) {
-    ut_ad(!table->is_temporary());
```

Comment:
I think this change is not needed. There was a bug because of which all tables were marked as temporary - https://github.com/gl-sergei/percona-xtrabackup/pull/17/files#diff-390ee556acee4b525d21b553f2a7be09R465